RI-SE/ATOS
# Feature c make create directories

This change enables us to do "make install" and all necessary directories are created for us, and executables are placed in system locations. No symlinks are created yet.

To test this, in the Maestro dir just do
`mkdir build && cd build`
and then
`cmake ..`
followed by
`make && sudo make install`
and the installed files should show up in /usr/local/bin, /usr/local/include and /usr/local/lib
